tool/vfx/vfxVkSection.h

```diff
@@ -361,6 +361,9 @@ class SectionPipelineOption : public Section {
     INIT_STATE_MEMBER_NAME_TO_ADDR(SectionPipelineOption, threadGroupSwizzleMode, MemberTypeEnum, false);
     INIT_STATE_MEMBER_NAME_TO_ADDR(SectionPipelineOption, reverseThreadGroup, MemberTypeBool, false);
     INIT_MEMBER_NAME_TO_ADDR(SectionPipelineOption, m_extendedRobustness, MemberTypeExtendedRobustness, true);
+#if VKI_RAY_TRACING
+    INIT_STATE_MEMBER_NAME_TO_ADDR(SectionPipelineOption, internalRtShaders, MemberTypeBool, false);
+#endif
     VFX_ASSERT(tableItem - &m_addrTable[0] <= MemberCount);
   }
```

Comment:
MemberCount probably needs an update?

Reply:
You would think so, but I counted and the MemberCount matches without an update.
So there was +1 in there already...

Reply:
There may be some internal-only things that add members as well!

Reply:
I just checked, there is indeed an internal-only member. Due to the way porting is done, I think it makes sense to have the same `MemberCount` here and internally (at least that’s how it has been handled until now). Otherwise we’ll get failures when porting this change to the internal branch (or even worse, no failures when assertions are not checked).

Reply:
Yes, I have fixed this up internally.

Reply:
Thanks!